src/Microsoft.Azure.SignalR.Management/Configuration/ServiceManagerOptions.cs

```diff
@@ -41,8 +42,13 @@ public class ServiceManagerOptions
         /// </summary>
         public ServiceTransportType ServiceTransportType { get; set; } = ServiceTransportType.Transient;
 
-        internal string ProductInfo { get; set; }
+        /// <summary>
+        /// Gets the json serializer settings that will be used to serialize content sent to Azure SignalR Service.
+        /// </summary>
+        public JsonSerializerSettings JsonSerializerSettings { get; } = new JsonSerializerSettings();
```

Comment:
also allow `set` following the convention of other properties?

Reply:
I didn't add a setter because there shouldn't be a need to set a new instance. I did it the same way like in https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.mvcnewtonsoftjsonoptions.serializersettings as developers should be used to that.

And what about the failed test? Do you agree that it doesn't have anything to do with my changes?

Reply:
The test failed randomly, already rerun and passed.